jest/react-native-device-info-mock.js

```diff
@@ -122,6 +122,7 @@ const diMock = {
   getBundleId: stringFnSync(),
   getDeviceId: stringFnSync(),
   getDeviceType: stringFnSync(),
```

Comment:
This one needs to be stringFnAsync now I think?
```suggestion
  getDeviceType: stringFnASync(),
```

Reply:
The original function is also not async, so I guess it's fine like this.

See https://github.com/react-native-device-info/react-native-device-info/blob/master/src/index.ts#L565

Reply:
Sorry for the delay! You are correct